src/BloomExe/Browser.cs

```diff
@@ -350,10 +357,16 @@ protected override void OnLoad(EventArgs e)
 			GeckoPreferences.User["layout.spellcheckDefault"] = 0;
 
 			RaiseGeckoReady();
-	   }
+
+			Debug.Assert(ParentForm != null);
+			ParentForm.Activated += (sender, ee) => _updateCommandsTimer.Enabled = true;
+			ParentForm.Deactivate += (sender, ee) => _updateCommandsTimer.Enabled = false;
```

Comment:
I'm not clear why we need this. Will we not get an enter event for the focused control when its window is activated? Won't this produce the situation you are trying to avoid...multiple browsers contending to control the Undo thing...whenever there is more than one on a form? If we need something on the form, maybe it should somehow only enable the timer for the focused child? Of course it may not be clear which child will end up with the focus at the point where the parent is activated...